include/ubiquity_motor/motor_hardware.h

```diff
@@ -45,9 +45,16 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <diagnostic_updater/update_functions.h>
 #include <diagnostic_updater/diagnostic_updater.h>
 
-#include <ubiquity_motor/motor_parmeters.h>
+#include <ubiquity_motor/motor_parameters.h>
 #include <ubiquity_motor/motor_serial.h>
 
+// To access I2C we need some system includes
+#include <linux/i2c-dev.h>
+#include <fcntl.h>
+#include <sys/ioctl.h>
+#define  I2C_DEVICE  "/dev/i2c-1"     // This is specific to default Magni I2C port on host
+#define  I2C_PCF8574_8BIT_ADDR  (uint8_t)0x40  // I2C addresses are 7 bits but often shown as 8-bit
```

Comment:
Its funny but I agree that these are best in the implementation file.  I will indeed move those to the src and use constants at the same time.  The history is I have a big I2C file and these were peeled off of that and put here because to pull in the entire thing would have meant a bigger project so I put in just what is required.  Granted not the 'ideal' case but by far the most efficient since I also changed everything to ROS_ logs .